packages/clay-label/src/ClayLabel.tsx

```diff
@@ -0,0 +1,57 @@
+/**
+ * © 2019 Liferay, Inc. <https://liferay.com>
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import * as React from 'react';
+import classNames from 'classnames';
+
+type DisplayType = 'secondary' | 'info' | 'warning' | 'danger' | 'success';
+
+interface Props
+	extends React.BaseHTMLAttributes<HTMLAnchorElement | HTMLSpanElement> {
+	closeButtonProps?: React.ButtonHTMLAttributes<HTMLButtonElement>;
+	displayType?: DisplayType;
+	large?: boolean;
+}
+
+const ClayLabel: React.FunctionComponent<Props> = ({
+	children,
+	className,
+	closeButtonProps,
+	displayType = 'secondary',
+	href,
+	large = false,
+	...otherProps
+}) => {
+	const TagName = href ? 'a' : 'span';
+
+	return (
+		<TagName
+			{...otherProps}
+			className={classNames('label', className, {
+				'label-dismissible': closeButtonProps,
+				'label-lg': large,
+				[`label-${displayType}`]: displayType,
+			})}
+			href={href}
+		>
+			<span className="label-item label-item-expand">{children}</span>
+
+			{closeButtonProps && (
+				<span className="label-item label-item-after">
+					<button
+						{...closeButtonProps}
+						className="close"
+						type="button"
+					>
+						{closeButtonProps.children}
```

Comment:
hey @bryceosterhaus, since we have a `closeButtonProps` API I think it makes sense to use` ClayIcon` directly here to live up to its name and usage.

Reply:
Good call, I didn't think about that. Just updated